README.md

```diff
@@ -190,24 +206,28 @@ You can use this flake in other flakes or Nix derviations.
 
 ### Visual Studio Code:
 
-Run `npm install && npm run compile` in `editor/code`.
+Run `npm install` in `editor/code`.
 
 ```sh
-(cd editor/code && npm i && npm run compile)
+(cd editor/code && npm i)
```

Comment:
We can do `make extension`. Could you also update the makefile target there to the sensible command?

Reply:
If you develop the extension with VS Code, make is automatic, I'll clarify this indeed.

Reply:
I meant that we recommend (cd editor/code && npm i) and we should do that in the makefile too.

Reply:
I don't use the makefile for the extension, so I'm unsure what you need there. There are a few different build targets for the extension so I'm unsure which one is the right one for make. We have:
- `vscode:prepublish`: a release, build, minified, and lighter.
- `esbuild`: fast transpiling step, for the extension and webview, includes dev stuff such as sourcemaps
- `typecheck`: typecheck the code, the IDE does that by itself anyways so that's not used
- `watch`: usual target that VS Code runs for extension development